docs/content/scheduler.md

```diff
@@ -118,6 +118,19 @@ learning_rate = 0.7
 momentum = 0.9
 ```
 
+**Upload Trained Model To HF**:
+To upload the trained model to Hugging Face provide a repository and a token with with the correct rights:
+
+```toml
+[scheduler.job.model_destination]
+repository="repository/identifier"
+token="hf_*"
```

Comment:
Let's add a note/warning that the token is going to be shared with the workers - same warning should be added to using a token for fetching a model.